applications/pelion_client/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+
+if (NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE ZDebug)
+endif()
+
+if (CONFIG_NET_L2_OPENTHREAD)
+  set(OT_CONFIG "${CMAKE_CURRENT_SOURCE_DIR}/configuration/common/ot_config.h")
+endif()
+
+if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
+  message(FATAL_ERROR
+          "Board ${BOARD} is not supported.\n"
+          "Please make sure board specific configuration files are added to "
+          "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
+endif()
+
+if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_${CMAKE_BUILD_TYPE}.conf")
+  message(FATAL_ERROR
+          "Configuration file for build type ${CMAKE_BUILD_TYPE} is missing.\n"
+          "Please add file ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_${CMAKE_BUILD_TYPE}.conf")
+endif()
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/dts.overlay")
+  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/dts.overlay")
+endif()
+
+set(mcuboot_CONF_FILE
+  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot_${CMAKE_BUILD_TYPE}.conf
+  )
+
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/spm_${CMAKE_BUILD_TYPE}.conf)
+  set(spm_CONF_FILE
+    ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/spm_${CMAKE_BUILD_TYPE}.conf
+    )
+endif()
```

Comment:
Same child image conf comment.

Reply:
accepted as-is for current time.
Let's continue discussions on way forward, and adopt all apps to the final decision.